nextcloudpi.sh

```diff
@@ -120,7 +120,7 @@ sleep 0.5 && echo "" > /run/ncp.log
 exit $RET
 EOF
   chmod 700 /home/www/ncp-launcher.sh
-  echo "www-data ALL = NOPASSWD: /home/www/ncp-launcher.sh , /sbin/halt" >> /etc/sudoers
+  echo "www-data ALL = NOPASSWD: /home/www/ncp-launcher.sh , /sbin/halt, /sbin/reboot" >> /etc/sudoers
 
```

Comment:
this needs to be included too for old images, so it should be duplicated in `update.sh`

Reply:
Ah right. I thought I had already added this. :+1:

Reply:
you added it in nextcloudpi.sh which is only used for building new images. If we are to update currently running images with the new feature through `ncp-update`, then we need to do this in `update.sh`

other changes, like any changes to files in `bin`, `ncp-web` and `etc` are overwritten automatically without more intervention, but this is `/etc/sudoers` so we have to do it explicitly.

Reply:
Yes, I thought I had added it twice. ;)
But apparently not so.

Reply:
:D looking great, once those two little things are fixed, I will build a devel image with `build_devel.sh` and test it.

I want to make a command to update from devel, so we don't need to use QEMU and generate images to test.